crates/torii/core/src/sql.rs

```diff
@@ -1175,9 +1198,383 @@ impl Sql {
+                "INSERT INTO tokens (id, contract_address, name, symbol, decimals) VALUES (?, ?, \
+                 ?, ?, ?)",
+                vec![
+                    Argument::String(token_id.clone()),
+                    Argument::FieldElement(contract_address),
+                    Argument::String(name),
+                    Argument::String(symbol),
+                    Argument::Int(decimals.into()),
+                ],
+            );
+        }
+
+        // Insert transfer event to erc721_transfers table
+        {
+            let insert_query = "INSERT INTO erc721_transfers (contract_address, from_address, \
+                                to_address, token_id, executed_at) VALUES (?, ?, ?, ?, ?)";
+
+            self.query_queue.enqueue(
+                insert_query,
+                vec![
+                    Argument::FieldElement(contract_address),
+                    Argument::FieldElement(from),
+                    Argument::FieldElement(to),
+                    Argument::String(token_id.clone()),
+                    Argument::String(utc_dt_string_from_timestamp(block_timestamp)),
+                ],
+            );
+        }
+
+        // Update balances in erc721_balances table
+        {
+            let update_query = "
+            INSERT INTO balances (id, balance, account_address, contract_address, token_id)
+            VALUES (?, ?, ?, ?, ?)
+            ON CONFLICT (account_address, contract_address, token_id) 
+            DO UPDATE SET balance = excluded.balance";
+
+            if from != Felt::ZERO {
+                self.query_queue.enqueue(
+                    update_query,
+                    vec![
+                        Argument::String(format!("{:#x}:{}", from, &token_id)),
+                        Argument::FieldElement(from),
+                        Argument::FieldElement(contract_address),
+                        Argument::String(u256_to_sql_string(&U256::from(0u8))),
+                        Argument::String(token_id.clone()),
+                    ],
+                );
+            }
+
+            if to != Felt::ZERO {
+                self.query_queue.enqueue(
+                    update_query,
+                    vec![
+                        Argument::String(format!("{:#x}:{}", to, &token_id)),
+                        Argument::FieldElement(to),
+                        Argument::FieldElement(contract_address),
+                        Argument::String(u256_to_sql_string(&U256::from(1u8))),
+                        Argument::String(token_id.clone()),
+                    ],
+                );
+            }
+        }
+        self.query_queue.execute_all().await?;
+
+        Ok(())
+    }
 }
 
 fn felts_sql_string(felts: &[Felt]) -> String {
     felts.iter().map(|k| format!("{:#x}", k)).collect::<Vec<String>>().join(FELT_DELIMITER)
         + FELT_DELIMITER
 }
+
+fn u256_to_sql_string(u256: &U256) -> String {
+    format!("{:#064x}", u256)
```

Comment:
change this to `066`

Reply:
discussion here: https://github.com/dojoengine/dojo/pull/2277#discussion_r1729458764